src/components/LogInComponent/LogInComponent.jsx

```diff
@@ -1,48 +1,98 @@
+
+  const handleLogout = () => {
+    dispatch(setUser({}));
+    cookies.remove("token");
+    cookies.remove("name");
+    cookies.remove("last_name");
+    cookies.remove("email");
+    toast.success(languageSelector(language, "logOutSuccess"));
+  };
+  const isUserLoggedIn = Object.keys(user).length;
+  const isAdmin = user.user_role === "restaurantAdmin" || user.user_role === "appAdmin";
 
   return (
-    <div className="log-in-component">
-      <div
-        className="log-in-component-background"
-        onClick={() => setShowLogIn(false)}
-      ></div>
-      <div className="log-in-form">
-        <div className="log-in-form-title-box">
-          <span className="log-in-main-title">{languageSelector(language, 'logInTitle')}</span>
-          <span className="log-in-main-subtitle">
-            {languageSelector(language, 'logInSubtitle')}
-          </span>
-        </div>
-        <div className="log-in-form-main-form">
-          {whichForm ? (
-            <LoginFormComponent setWhichForm={setWhichForm} closeModal={() => setShowLogIn(false)}/>
-          ) : (
-            <ForgottenPasswordComponent setWhichForm={setWhichForm} />
-          )}
-        </div>
-        <div className="log-in-sign-up">
-          <span
-            className="log-in-sign-up-text"
-            onClick={() => navigate("/register")}
-          >
-            {languageSelector(language, 'logInRegister')}
-          </span>
+    <>
+      <ToastContainer/>
+      <div className="log-in-component">
+        <div
+          className="log-in-component-background"
+          onClick={() => setShowLogIn(false)}
+        ></div>
+        <div className="log-in-form">
+          <div className="log-in-form-title-box">
+            <span className="log-in-main-title">{languageSelector(language, 'logInTitle')}</span>
+            <span className="log-in-main-subtitle">
+              {languageSelector(language, 'logInSubtitle')}
+            </span>
+          </div>
+          <div className="log-in-form-main-form">
+            {isUserLoggedIn ? (
+                <div className='log-in-main-form-buttons-container'>
+                  <button className='log-in-main-form-buttons' onClick={() => navigate('/user')}>
+                    {languageSelector(language, 'userPage')}
+                  </button>
+                  {isAdmin && (
+                    <button
+                      className="log-in-main-form-buttons"
+                      onClick={() =>
+                        navigate(
+                          user.user_role === "restaurantAdmin"
+                          ? "/restaurant-admin"
+                          : "/general-admin"
+                        )
+                      }
+                    >
+                    {languageSelector(
+                      language,
+                      user.user_role === "restaurantAdmin"
+                      ? "restaurantAdminPage"
+                      : "appAdminPage"
```

Comment:
same here.

Reply:
These are for the language they are not routes, should I still change them?

Reply:
let's keep it there then, np.